pkg/deploy/deploy.go

```diff
@@ -173,7 +178,38 @@ func (d *deployer) Deploy(ctx context.Context) error {
 		}
 	}
 
-	return nil
+	return d.saveRPVersion()
+}
+
+// saveRPVersion for current location in shared storage account for environment
+func (d *deployer) saveRPVersion() error {
+	keys, err := d.accounts.ListKeys(context.Background(), *d.config.Configuration.GlobalResourceGroupName, *d.config.Configuration.RPVersionStorageAccountName, "")
+	if err != nil {
+		return err
+	}
+	key := *(*keys.Keys)[0].Value
+
+	storageClient, err := azstorage.NewBasicClient(*d.config.Configuration.RPVersionStorageAccountName, key)
+	if err != nil {
+		return err
+	}
+
+	blobClient := storageClient.GetBlobService()
+	// one container per RP location
+	containerRef := blobClient.GetContainerReference(d.config.Location)
+	_, err = containerRef.CreateIfNotExists(nil)
+	if err != nil {
+		return err
+	}
+
+	// gets version blob
+	blobRef := containerRef.GetBlobReference("version")
+	b, err := json.Marshal(d.version)
+	if err != nil {
+		return err
+	}
+	// creates version blob block with current version in the container named after the location
```

Comment:
Is this comment true? If this is running in eastus and after this germanywestcentral, will it not override the value? And we should store location name too.

Reply:
Yes, I think that using separate containers would be sufficient to isolate the blobs. Location name is already stored as the container's name.